src/main/java/org/spongepowered/api/block/data/Sign.java

```diff
@@ -37,7 +37,7 @@
      *
      * @return The lines of text
      */
-    Message[] getLines();
+    Text[] getLines();
```

Comment:
It wasnt my intention to bother you. Sorry for that.
I probably have missed that there.